graphql_compiler/cost_estimation/statistics.py

```diff
@@ -97,16 +115,39 @@ def get_field_quantiles(self, vertex_name, field_name):
         """
         return None
 
+    def get_value_count(self, vertex_name: str, field_name: str, value: Any) -> Optional[int]:
+        """Return the estimated number of times the given value appears in the database.
+
+        Args:
+            vertex_name: vertex on which the field is defined
+            field_name: field for which the value stands
+            value: value to be counted
+
+        Returns:
+            An estimate on how often this value currently appears in the given field, or None
+            if unkwnown.
+        """
+        return None
+
 
 class LocalStatistics(Statistics):
     """Statistics class that receives all statistics at initialization, storing them in-memory."""
 
+    # See __init__ docstring for definitions.
+    _class_counts: Dict[str, int]
+    _vertex_edge_vertex_counts: Dict[Tuple[str, str, str], int]
+    _distinct_field_values_counts: Dict[Tuple[str, str], int]
+    _field_quantiles: Dict[Tuple[str, str], List[Any]]
+    _sampling_summaries: Dict[str, SamplingSummary]
+
     def __init__(
         self,
-        class_counts,
-        vertex_edge_vertex_counts=None,
-        distinct_field_values_counts=None,
-        field_quantiles=None,
+        class_counts: Dict[str, int],
+        *,
+        vertex_edge_vertex_counts: Optional[Dict[Tuple[str, str, str], int]] = None,
+        distinct_field_values_counts: Optional[Dict[Tuple[str, str], int]] = None,
+        field_quantiles: Optional[Dict[Tuple[str, str], List[Any]]] = None,
+        sampling_summaries: Optional[Dict[str, SamplingSummary]] = None,
```

Comment:
I think the need to fix the test below is because of this: the `*` means that `distinct_field_value_counts` is now kwarg-only, i.e. the test needed fixing. Mypy found it via static analysis, but this would have also failed at runtime.

Reply:
Mypy found if when I just added type hints, which reminded me to add `*`